docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -4151,6 +4151,13 @@
               }
             }
           },
+          "other-concurrent-attempts" : {
+            "type" : "array",
+            "items" : {
+              "type" : "object",
+              "$ref" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:job:SubtaskExecutionAttemptDetailsInfo"
+            }
+          },
```

Comment:
Changes in `rest_v1_dispatcher.html` and `rest_v1_dispatcher.yml` seem unrelated. I assume these are fixes for previous changes that forget to regenerate the open-api & doc files. Therefore, they should be placed in a separate hotfix commit.

Reply:
Done.